### DISCUSSION TITLE
Is the 24 hr YouTube course still up-to-date enough to be worthy of the time investment?

### DISCUSSION THREAD

**Discussion by ghost** [mrdbourke/pytorch-deep-learning#899] [Q&A]
I'm new to ML/DL but very experienced with Python. TIA

**Comment by Nipuncp** [comment]
I tried the code. It still works

**Comment by LuluW8071** [comment]
Yes, it still works. However, the video is just a basic on PyTorch. You have to do more projects and research yourself if you want more advance concepts.
